pkg/proc/bininfo.go

```diff
@@ -192,7 +192,13 @@ func FindFileLocation(p Process, filename string, lineno int) ([]uint64, error)
 	}
 
 	if len(pcs) == 0 {
-		return nil, &ErrCouldNotFindLine{fileFound, filename, lineno}
+		stripped := false
+		for _, image := range bi.Images {
+			if image.Stripped() && image.symTable != nil && image.symTable.Files[filename] != nil {
+				stripped = true
```

Comment:
Not that the amount of images would likely be a performance issue, but we can break after this assignment, right?

Reply:
Done.